### DISCUSSION TITLE
Welcome to 🔥 Maester  💬 Discussions and 💡 Ideas!

### DISCUSSION THREAD

**Discussion by merill** [maester365/maester#264] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of the Maester community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Haakonak** [comment]
Hi! I am very new to Maester and began implementing it and running the tests on my tenant, but i have noticed that all tests seem to have run twice. I am wondering if this is intended or if something was done wrong on my end. I have attached a screenshot of what i am seeing in the logs. 

![MultipleTests](https://github.com/maester365/maester/assets/79579888/e064a1c8-086f-4780-9f95-190458f5f9df)

**Reply by simon-barratt-seq** [reply]
I see the same behaviour

**Reply by Haakonak** [reply]
I see that adding the lines of code shown in the bottom of the README seem to have fixed this issue.

The lines of code:
        include_public_tests: true # Optional
        pester_verbosity: None # Optional - 'None', 'Normal', 'Detailed', 'Diagnostic'
        
I have set the "include_public_tests" to false. You can give it a try and see if this also fixes the issue for you @simon-barratt-seq
